src/fraction.ts

```diff
@@ -831,6 +833,20 @@ export class Fraction {
     }
   }
 
+  /**
+   * Check if two {@link Fraction} instances represent the same rational number.
+   * Examples:
+   * ```ts
+   * new Fraction("19.7").strictEquals(new Fraction("98/5")) // false
+   * new Fraction("19.6").strictEquals(new Fraction("98/5")) // true
+   * new Fraction("19.5").strictEquals(new Fraction("98/5")) // false
+   * ```
+   **/
+  strictEquals(other: Fraction) {
```

Comment:
This one fails when using heterogenous `HashMap`s with other non-primitive keys.